cbopensource/connectors/threatconnect/config.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import sys
+
+_logger = logging.getLogger(__name__)
+
+
+class Config(object):
+    # noinspection PyUnusedName
+    feed_name = "threatconnectintegration"
+    # noinspection PyUnusedName
+    display_name = "ThreatConnect"
+    cb_image_path = "/carbonblack.png"
+    integration_image_path = "/threatconnect.png"
+    # noinspection PyUnusedName
+    integration_image_small_path = "/threatconnect-small.png"
+    json_feed_path = "/threatconnect/json"
+    directory = "/usr/share/cb/integrations/cb-threatconnect-connector/content"
+
+    def __init__(self, config_options):
+        self._options = config_options
+        self._errors = []
+        self.debug = self._get_boolean('debug')
+        self.log_level = self._get_string('log_level', "INFO", valid=["DEBUG", "INFO", "WARNING", "ERROR"],
+                                          coerce=True, to_upper=True)
+        self.log_file_size = self._get_int('log_file_size', 10 * 1024 * 1024,
```

Comment:
another opportunity to use either a class variable or a constant (for the size)

Reply:
Fixed with MiB constant.